core/app/models/spree/promotion.rb

```diff
@@ -64,9 +64,24 @@ def as_json(options = {})
       super
     end
 
+    def not_started?
+      starts_at.present? && starts_at > Time.current
+    end
+
+    def started?
+      !not_started?
```

Comment:
I think I would switch where the logic resided so `started?` contained the time check and `not_started?` could be `!started?` which reads a bit better. As it's written here it's sort of a double-negative for `started?` being "not not_started?"

Reply:
I have applied the change @ericsaupe thanks for the feedback. 

What do you think 😄 ?